webapp/src/filelist.tsx

```diff
@@ -362,11 +362,13 @@ export class FileList extends data.Component<ISettingsProps, FileListState> {
                 return Promise.resolve()
             }
             let fileText = "";
-            if (fileName == customFile) {
+            const tsFiles = pkgCfg.files.filter(f => f.endsWith(".ts"));
```

Comment:
Ah wait, have you checked this in arcade? We have some generated files over there with the `.g.ts` file extension (e.g. add  a tilemap to your project in blocks and then convert to ts)

Reply:
I haven't. I'll test it there with a tilemap now.

Reply:
Yeah, it looks like tilemaps break this. Good catch. Is it okay to just add a case to filter out `.g.ts` files? Or are there other main package files that I should look out for?

Reply:
Talked offline about this, just going to filter out `.g.ts` files from the included files.